src/python/dxpy/bindings/dxrecord.py

```diff
@@ -82,6 +82,8 @@ def _new(self, dx_hash, **kwargs):
         :type dx_hash: dict
         :param init_from: Record from which to initialize the metadata
         :type init_from: :class:`DXRecord`
+        :param close: Whether or not to close the record immediately after creating it
```

Comment:
I think it's more appropriate for this to be explicitly named as a keyword arg to the `_new` method. The `media_type` argument for `DXFile` object might be a useful precedent here, since it's an option specific to files, just as `close` is an option specific to records.